### ISSUE TITLE
Are there any benchmarks?

### ISSUE THREAD

**Issue by MaxGraey** [fitzgen/dodrio#38]
It will be great have some one on this site:
https://stefankrause.net/js-frameworks-benchmark8/table.html

**Comment by rumanbsl** [comment]
https://hacks.mozilla.org/2019/03/fast-bump-allocated-virtual-doms-with-rust-and-wasm/

**Comment by MaxGraey** [comment]
I saw this: https://hacks.mozilla.org/2019/03/fast-bump-allocated-virtual-doms-with-rust-and-wasm


Library | Optimized? | Milliseconds
-- | -- | --
Angular 2 | Yes | 2371
Dodrio 0.1-prerelease | No | 2181

But Angular is not fastest framework. It will be great see comparison with **vanilla js** (as baseline) and **stdweb** .

**Comment by liamcurry** [comment]
FYI I created an implementation of that benchmark for `wasm-bindgen` that can be used as a point of reference/starting point: https://github.com/krausest/js-framework-benchmark/tree/master/frameworks/keyed/wasm-bindgen

It looks like `wasm-bindgen` is pretty fast! Currently it's ranked 4th https://krausest.github.io/js-framework-benchmark/current.html

Another benchmark worth mentioning is UIBench. https://localvoid.github.io/uibench/ I like this one because it's easier to implement and run benchmarks. Here's a `wasm-bindgen` implementation I created as a point of reference: 
- https://github.com/sagan-software/uibench-wasm-bindgen
- https://sagan-software.github.io/uibench-wasm-bindgen/

**Comment by MaxGraey** [comment]
Yes, pure `wasm-bindgen`  as well as `stdweb` pretty fast, but frameworks with vdom like **yew** (which use stdweb under the hood) is one of the slowest frameworks compared to JavaScript based

**Comment by liamcurry** [comment]
The `wasm-bindgen` benchmarks are ports of the vanilla JS implementations so they show the overhead costs vs JS.

Also there's nothing inherently wrong with VDOM. ivi uses VDOM and it's pretty fast. I imagine a Rust port would perform similarly, adjusting for overhead.

**Comment by fitzgen** [comment]
Looks like you've seen all the benchmarking that currently exists.

Working on an implementation for js-framework-benchmark, and will also see if that turns up any more interesting corners of perf to look at. Thanks for your patience :)

**Comment by krausest** [comment]
Looking forward to getting your PR. Let me know if you run into problems with the benchmark and I‘ll try to help.

**Comment by Th3Whit3Wolf** [comment]
Any updates on this? I'd love to see how dodrio performs in benchmarks vs other wasm projects and javascript frameworks.
